### PR TITLE
Rebase #59

### PR DESCRIPTION
**Pull request by bgruening**
This PR should rebase https://github.com/galaxyproject/tools-devteam/pull/59 so we can merge it.

### CODE REVIEW

**Comment by bgruening** [comment]
ping @martenson @nsoranzo @jsh58 
Sorry @jsh58 that I'm so slow.

File: tools/bowtie2/bowtie2_wrapper.xml [hunk 1]
```diff
@@ -160,18 +160,21 @@
...
             #if str( $analysis_type.other_options.other_options_selector ) == "yes":
+                ${analysis_type.other_options.reorder}
                 ${analysis_type.other_options.non_deterministic}
                 --seed "${analysis_type.other_options.seed}"
             #end if
         
         #elif str( $analysis_type.analysis_type_selector ) == "cline":
             ${analysis_type.cline}
-        #end if
+        #end if
```
Comment (nsoranzo) [inline tools/bowtie2/bowtie2_wrapper.xml]: Trailing whitespaces can be removed.

**Comment by bgruening** [comment]
@nsoranzo fixed.

**Comment by nsoranzo** [comment]
:+1:

**Comment by bgruening** [comment]
Fixes https://github.com/galaxyproject/tools-devteam/pull/59

**Pull request merged**